src/python/utils.py

```diff
@@ -2,19 +2,28 @@
 import os
 from bs4 import BeautifulSoup
 
+AUDIO_PATH = "audio/"
+IMAGE_PATH = "images/episodes/"
```

Comment:
Small nit but would you mind dropping the trailing slashes here and instead putting them down below where you actually build the path?

Reply:
Moved